src/tools/illink/test/Mono.Linker.Tests.Cases/Inheritance.Interfaces/DefaultInterfaceMethods/StaticDefaultInterfaceMethodOnStruct.cs

```diff
@@ -0,0 +1,45 @@
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+
+namespace Mono.Linker.Tests.Cases.Inheritance.Interfaces.DefaultInterfaceMethods
+{
+	[TestCaseRequirements (TestRunCharacteristics.SupportsDefaultInterfaceMethods, "Requires support for default interface methods")]
+	class StaticDefaultInterfaceMethodOnStruct
+	{
+		public static void Main ()
+		{
+#if SUPPORTS_DEFAULT_INTERFACE_METHODS
+			Foo<Derived> ();
+#endif
+		}
+
+#if SUPPORTS_DEFAULT_INTERFACE_METHODS
+		[Kept]
+		static void Foo<T> () where T : IBase
```

Comment:
```suggestion
		static void TestStaticDefaultMethodOnStruct<T> () where T : IBase
```

Or just `Test` - took me a while what it does... and it's just a way to call the method.